.github/workflows/check-generated-files.yml

```diff
@@ -51,12 +51,11 @@ jobs:
               - "pkg/messaging/pubsub.go"
               - "things/postgres/clients.go"
               - "things/things.go"
-              - "auth/authz.go"
-              - "auth/domains_client.go"
+              - "pkg/authz.go"
+              - "pkg/authn.go"
               - "auth/domains.go"
               - "auth/keys.go"
               - "auth/service.go"
-              - "auth/token_client.go"
```

Comment:
No need to remove this one as it should be checked

Reply:
But there is no file at this `auth/token_client.go`